### ISSUE TITLE
Coverage badge broken in repo readme

### ISSUE THREAD

**Issue by jedrazb** [Yelp/nrtsearch#400]
As in the title, the coverage badge seems to be broken, clicking the "Coverage" link gives 404:
<img width="340" alt="Screenshot 2021-11-11 at 11 28 06" src="https://user-images.githubusercontent.com/14121688/141290510-6845bdf1-019b-48d9-8b66-fc18b9623fd3.png">

**Comment by jedrazb** [comment]
@vim345 I see you were doing some work on coverage badge, is that expected for now?

**Comment by vim345** [comment]
@jedrazb Yes. Thanks for flagging it. I had disabled the configs, but I forgot to remove the reference point from README. I have fixed the configs in this PR: https://github.com/Yelp/nrtsearch/pull/405
